docs/framework-app_dynamics_agent.md

```diff
@@ -17,14 +19,14 @@ When binding AppDynamics using a user-provided service, it must have name or tag
 
 | Name | Description
 | ---- | -----------
```

Comment:
I believe we're still planning on supporting pre 4.1.  What was the release date for that?